api/bibliotheca.py

```diff
@@ -1288,9 +1282,35 @@ class BibliothecaEventMonitor(CollectionMonitor, TimelineMonitor):
     SERVICE_NAME = "Bibliotheca Event Monitor"
     DEFAULT_START_TIME = timedelta(365*3)
     PROTOCOL = ExternalIntegration.BIBLIOTHECA
+    LOG_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S'
 
     def __init__(self, _db, collection, api_class=BibliothecaAPI,
-                 cli_date=None, analytics=None):
+                 default_start=None, analytics=None,
+                 override_timestamp=False,
+                 ):
+        """Initializer.
+
+        :param _db: Database session object.
+
+        :param collection: Collection for which this monitor operates.
+
+        :param api_class: API class or an instance thereof for this monitor.
+        :param api-class: Union[Type[BibliothecaAPI], BibliothecaAPI]
```

Comment:
Is `api-class` meant to have the hyphen rather than the underscore?

Reply:
Typo. Thanks for catching that.